mmv1/third_party/terraform/main.go.erb

```diff
@@ -2,11 +2,56 @@
 package main
 
 import (
-	"github.com/hashicorp/terraform-plugin-sdk/v2/plugin"
+	"context"
+	"flag"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-go/tfprotov5"
+	"github.com/hashicorp/terraform-plugin-go/tfprotov5/tf5server"
+	"github.com/hashicorp/terraform-plugin-mux/tf5muxserver"
 	"github.com/hashicorp/terraform-provider-google<%= "-" + version unless version == 'ga'  -%>/google<%= "-" + version unless version == 'ga'  -%>"
+	ver "github.com/hashicorp/terraform-provider-google<%= "-" + version unless version == 'ga'  -%>/version"
+)
+
+var (
+	// these will be set by the goreleaser configuration
+	// to appropriate values for the compiled binary
+	version string = ver.ProviderVersion
```

Comment:
Hmm- I might be missing something here, how is this string getting used?

Reply:
ah! it's used by the plugin-framework version of the provider, i can take it out here and keep it just in the other PR